ProcessingShowPath/ProcessingShowPath.pde

```diff
@@ -9,19 +9,60 @@ float forwardTime;
 float backTime;
 int mistakes;
 int shortcuts;
+int startBiasX = -140;
+int startBiasZ = -50;
+int sizeX = 1200;
+int sizeY = 900;
+float stretchMultiplierX = 2.2;
+float stretchMultiplierY = 2.2;
+
+String[] filenames1;
+String[] filenames2;
+
+color forwardColor = color(0,0,255);
+color backColor = color(255,0,0);
+
+boolean drawForward = true;
+boolean drawBack = true;
 
 void setup() {
-  size(800,600);
-  PImage map = loadImage("minecraft.png");
+  size(1200,900);
+  PImage map = loadImage("isometric.png");
+  //map.filter(GRAY);
   background(map);
-  frameRate(1000000000);
-  reader = createReader("test.txt");
-  System.out.println("Started");
+  frameRate(30);
   loadPixels();  
-  c = color(0,0,255);
+  c = forwardColor;
+  println("Files for first condition: ");
+  filenames1 = listFileNames("C:/UnityProjects/P5Game/ProcessingShowPath/Condition1");
+  printArray(filenames1);
+
+  println("Files for second condition: ");
+  filenames2 = listFileNames("C:/UnityProjects/P5Game/ProcessingShowPath/Condition2");
+  printArray(filenames2);
```

Comment:
for loop?

Reply:
and have fileNames as a 2D array or do it with a list of arrays?

Reply:
nah

Reply:
why nah? It's bad way of coding like that

Reply:
all good, understand now that they mean conditions for Arrow and Bee, so they should just be renamed to make it better to understand and that's it 👍